pkg/controller/persistentvolume/persistentvolume_recycler_controller.go

```diff
@@ -270,6 +270,10 @@ func (f *PersistentVolumeRecycler) GetPodPluginDir(podUID types.UID, pluginName
 	return ""
 }
 
+func (f *PersistentVolumeRecycler) GetPodVolumesPluginDir(podUID types.UID, pluginName string) string {
```

Comment:
Looks like `GetPodVolumesPluginDir(...)` is a leftover from a previous iteration, and no longer needed. Can you please remove all instances from the PR.